lib/java/ydoc-server/src/main/java/org/enso/ydoc/Main.java

```diff
@@ -0,0 +1,43 @@
+package org.enso.ydoc;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import org.enso.ydoc.polyfill.ParserPolyfill;
+import org.enso.ydoc.polyfill.web.WebEnvironment;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.io.IOAccess;
+
+public class Main {
+
+  private static final String YDOC_SERVER_PATH = "/dist/assets/ydocServer.js";
+
+  private Main() {}
+
+  public static void main(String[] args) throws Exception {
+    System.setProperty("helidon.serialFilter.pattern", "javax.management.**;java.lang.**;java.rmi.**;javax.security.auth.Subject;!*");
+
+    var ydoc = Main.class.getResource(YDOC_SERVER_PATH);
+    var contextBuilder = WebEnvironment.createContext().allowIO(IOAccess.ALL);
+
+    Sampling.init();
```

Comment:
optional?

Reply:
Yes, optional. Just moved the logic to a separate class.